extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -79,6 +80,17 @@ private YoutubeParsingHelper() {
     private static final String[] HARDCODED_YOUTUBE_MUSIC_KEYS = {"AIzaSyC9XL3ZjWddXya6X74dJoCTL-WEYFDNX30", "67", "0.1"};
     private static String[] youtubeMusicKeys;
 
+    /**
+     * <code>PENDING+999</code> means that the user did not yet submit their choices.
+     * Therefore, YouTube & Google should not track the user,
+     * because they did not give consent. The three digits at the end can be random.
+     */
+    public static final String CONSENT_COOKIE_VALUE = "PENDING+999";
```

Comment:
if the three digits can be random, should we not consider randomizing them? This would make it harder to detect that the Extractor and not a regular browser is being used. Or are the three digits time based? e.g. 100 means that the user was asked shortly ago, while 999 means it was long ago? In that case I would stick with the 999.

Maybe you could add the source (url) to the explanation of that.

Reply:
Well, I did several tests with completing the form on my PC and always got different digits from `001` to `999`. I think there is no other public documentation than YouTube's privacy policy.